src/morphology.rs

```diff
@@ -688,13 +721,37 @@ impl Mask {
 /// # }
 /// ```
 pub fn grayscale_dilate(image: &GrayImage, mask: &Mask) -> GrayImage {
-    let result = GrayImage::from_fn(image.width(), image.height(), |x, y| {
-        Luma([mask
-            .apply(image, x, y)
-            .map(|l| l.0[0])
-            .max()
-            .unwrap_or(u8::MIN)]) // default is u8::MIN because it's the neutral element of max
-    });
+    let mut result = GrayImage::from_fn(image.width(), image.height(), |_, _| Luma([u8::MIN]));
+    for (y_16, x_line) in mask.lines() {
```

Comment:
I don't like postfixes, just shadow the variable

Reply:
done in next commit